bobcat/bb-aem-classic/src/main/java/com/cognifide/qa/bb/aem/ui/wcm/windows/CreateSiteWindow.java

```diff
@@ -96,7 +96,7 @@ public CreateSiteWindow waitToBeDisplayed() {
    * @return this CreateSiteWindow
    */
   public CreateSiteWindow typeTitle(String title) {
-    textField.setValue(title);
+    textFields.get(0).setValue(title);
```

Comment:
it's ugly but safe. Treat this as a workaround, I'll try to add the capability of injecting multiple page objects of the same type in scope of #57